vaadin-crud-flow-parent/vaadin-crud-flow-integration-tests/src/test/java/com/vaadin/flow/component/crud/tests/EventHandlingIT.java

```diff
@@ -193,7 +193,7 @@ public void emptyInvalidFieldsIndicatedOnSave() {
 
         crud.getEditorSaveButton().click();
 
-        Assert.assertTrue(firstNameField.hasAttribute("invalid"));
+        Assert.assertTrue(lastNameField.isInvalid());
```

Comment:
There is copy paste error in variable name, should be:

Assert.assertTrue(fistNameField.isInvalid());

Reply:
Changed